packages/resume/src/stylesheet/__fixtures__/v1/selectors.css

```
@@ -0,0 +1,13 @@
+/* biome-ignore lint/suspicious/noUnknownAtRules: RRSS version directive */
+@rr-version 1;
```

Stylelint (17.14.1)

[error] Unexpected unknown at-rule "`@rr-version`" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)